scene/main/scene_tree.cpp

```diff
@@ -1262,6 +1265,7 @@ void SceneTree::_process(bool p_physics) {
 
 				if (using_threads) {
 					local_process_group_cache.clear();
+					local_process_group_cache.reserve(i);
```

Comment:
I'm not sure if this is helpful or not. `local_process_group_cache` is cached so it isn't really expanding at run time anyway. Plus the loop runs from `j` to `i` so this reserve is always over-allocating memory.

Reply:
Well, it is being cleared every iteration before the for loop does multiple reallocations. 

I hadn't noticed the `j = from`, gonna fix it soon.